sky/server/requests/payloads.py

```diff
@@ -32,12 +32,19 @@
 
 logger = sky_logging.init_logger(__name__)
 
+# These non-skypilot environment variables will be updated from the local
+# environment on each request.
+external_env_vars = [
+    'AWS_PROFILE',
```

Comment:
Since we are adding this, there is actually a few other env vars supported by AWS, which was reported to be used: https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-envvars.html

```console
$ export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
$ export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
```